src/ankr/handlerFactory.ts

```
@@ -94,7 +92,7 @@ export function createAnkrHandler<TRequest, TResponse>({
     runtime: IAgentRuntime,
     message: Memory,
     state?: State,
-    options: { [key: string]: unknown } = {},
+    options?: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)